CHANGELOG.md

```diff
@@ -1,20 +1,21 @@
-## Parity [v1.11.0](https://github.com/paritytech/parity/releases/tag/v1.10.0) (2018-05-09)
+## Parity [v1.11.1](https://github.com/paritytech/parity/releases/tag/v1.11.1) (2018-05-15)
 
-This is the Parity 1.11.0-beta release! Hurray!
+This is the Parity 1.11.1-beta release! Hurray!
 
 Notable changes in reversed alphabetical order:
 
 - TOOLING: **Whisper CLI** [#8201](https://github.com/paritytech/parity/pull/8201)
   - `whisper-cli` is a standalone tool to communicate with the Whisper protocol.
   - It provides functionality to specify `whisper-pool-size`, `port` and `address` to use.
   - All whisper RPC APIs are enabled and can be directly acessed.
+I'm not used to writing these changelogs but I guess that would explain it
```

Comment:
Remove :smiley:

Reply:
jesus why do I keep copy-pasting this in xD